### PR TITLE
replaced 'add_to_river' with 'elgg_create_river_item' for new videos

### PR DESCRIPTION
**Pull request by propertunist**
the 1st of my updates to bring videolist to 1.11.x compatibility

### CODE REVIEW

File: manifest.xml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 	<category>multimedia</category>
 	<description>This plugin allows users to create a library of videos.</description>
 	<website>http://lorea.cc/</website>
-	<copyright>(C) prateek.choudhary 2008-09, Elgg 2010, Lorea 2011</copyright>
+	<copyright>(C) prateek.choudhary 2008-09, Elgg 2010, Lorea 2011, ureka 2015</copyright>
 	<license>GNU Public License version 2</license>
 	<requires>
 		<type>elgg_release</type>
-		<version>1.8</version>
+		<version>1.11</version>
```
Comment (mrclay) [inline manifest.xml]: Does this really need past 1.9?

File: actions/videolist/edit.php [hunk 2]
```diff
@@ -93,7 +93,11 @@
 	system_message(elgg_echo('videolist:saved'));
 
 	if ($new_video) {
-		add_to_river('river/object/videolist_item/create', 'create', elgg_get_logged_in_user_guid(), $video->guid);
+            elgg_create_river_item(array(
```
Comment (mrclay) [inline actions/videolist/edit.php]: Need tabs instead of spaces

File: actions/videolist/edit.php [hunk 3]
```diff
@@ -93,7 +93,11 @@
...
+                'view' => 'river/object/videolist_item/create', 
+                'action_type' => 'create', 
+                'subject_guid' => elgg_get_logged_in_user_guid(), 
+                'object_guid' => $video->guid));
```
Comment (mrclay) [inline actions/videolist/edit.php]: All array elements on their own line, with trailing comma.
```php
        'object_guid' => $video->guid,
));
```

File: manifest.xml (hunk 1 above)
Comment (propertunist) [inline manifest.xml]: probably not; i unconsciously want everyone to upgrade ;)

File: actions/videolist/edit.php (hunk 2 above)
Comment (propertunist) [inline actions/videolist/edit.php]: ok, what is the easiest way for me to make that change?

Comment (mrclay) [inline actions/videolist/edit.php]: I have Elgg projects set up in PHPStorm to use tabs, and I can just select some code, un-indent/re-indent.

**Comment by mrclay** [comment]
BTW, here's what I'd do: `git reset --soft 06acdb78f4cf4352caf9f46ec82b2e08f926084d` then make changes and commit. That soft reset will leave files as is but remove your commits. After making a new one, force push.

**Comment by propertunist** [comment]
ok, well, i navigated to the local repository using the terminal here and rand the command you shared here.. but i think i need to add a reference to the github repo too, don't i?
i've only used visual git tools before, so could do with a nudge.
do i need to add: --git-dir=<path>
followed by the remote address of my forked repo?

**Comment by mrclay** [comment]
You don't need a reference to the official repo. The command I gave should erase your commits, but leave the code. Then you make your changes and commit.

**Comment by propertunist** [comment]
when i run that command in the terminal (while located at the local project folder for my fork of videolist) i see: 
```
fatal: Could not parse object '06acdb78f4cf4352caf9f46ec82b2e08f926084d'
```

**Comment by mrclay** [comment]
It might mean your local master branch isn't up to date with the official repo. You can see that commit is the current one: https://github.com/Elgg/videolist/commits/master.

Here's how I'd fix that:
```
git remote add upstream https://github.com/Elgg/videolist.git
git fetch upstream
git reset --soft upstream/master
```

**Comment by propertunist** [comment]
aha ok, thanks - yes my fork is behind the main elgg repository for videolist.
i ran the 3 commands you gave here and also the original reset one afterwards. there were no error messages so i thought it all worked ok. when i view the github page for my fork, it now shows that my fork is behind the master elgg repo with some changes and is ahead with my own changes..
so as far as i can discern github is not aware that i have intended to update my local repo with the full changes from the main elgg repo.
also, i 'think' that the netbean git viewer's history page is showing that i have integrated the recent changes from elg/videolist master brandch locally - but i am not sure. the evidence that i have not done so is that i can still see the youtube.php file in my local repo and that has been removed by one of your recent changes.
all of this is why i have avoided git for so long! i have not yet found a clear way to use it without being confused by the lack of oversight of what is occurring.
any thoughts/tips welcome!

**Comment by mrclay** [comment]
Don't worry about how your fork looks on GitHub. All that matters is that your work branch is up to date. The problem I didn't notice is that you're working on master. Don't do that :) I've just updated these pages a bit. Have a look. https://community.elgg.org/pages/view/1123538/1-elgg-bug-fixing-with-git

**Pull request closed without merging**